gbfs-validator/versions/schemas/v3.0-RC/gbfs_versions.json

```
@@ -0,0 +1,65 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id":
+    "https://github.com/NABSA/gbfs/blob/v3.0-RC/gbfs.md#gbfs_versionsjson-added-in-v11",
+  "description": "Lists all feed endpoints published according to version sof the GBFS documentation. (added in v1.1)",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description":
+        "Last time the data in the feed was updated in POSIX time.",
+      "type": "integer",
+      "minimum": 1450155600
+    },
+    "ttl": {
+      "description":
+        "Number of seconds before the data in the feed will be updated again (0 if the data should always be refreshed).",
+      "type": "integer",
+      "minimum": 0
+    },
+    "version": {
+      "description":
+        "GBFS version number to which the feed conforms, according to the versioning framework.",
+      "type": "string",
+      "const": "3.0-RC"
+    },
+    "data": {
+      "description": "Response data in the form of name:value pairs.",
+      "type": "object",
+      "properties": {
+        "versions": {
+          "description":
+            "Contains one object, as defined below, for each of the available versions of a feed. The array must be sorted by increasing MAJOR and MINOR version number.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "version": {
+                "description": "The semantic version of the feed in the form X.Y",
+                "type": "string",
+                "enum": [
+                  "1.0",
+                  "1.1",
+                  "2.0",
+                  "2.1",
+                  "2.2",
+                  "2.3",
+                  "3.0-RC"
```

Let's add v3.0 in v3.0-RC like in https://github.com/MobilityData/gbfs-json-schema/pull/87. Thank you!

```suggestion
                  "3.0-RC",
                  "3.0"
```

---

I've updated the files from https://github.com/MobilityData/gbfs-validator/pull/104#pullrequestreview-1491739949 so this is now included.